src/apollo/ApolloCoordinator.sol

```diff
@@ -20,26 +20,28 @@ contract ApolloCoordinator is IApolloCoordinator {
      * @param dataFeedId The identifier of the data feed being requested.
      * @param callbackGasLimit The gas limit for the callback transaction.
      */
-    function requestDataFeed(string memory dataFeedId, uint256 callbackGasLimit) public {
-        uint256 id = _getRequestId();
-        requests[id] = PriceFeedRequest(id, dataFeedId, callbackGasLimit, msg.sender);
-        emit PriceFeedRequested(id, dataFeedId, callbackGasLimit, msg.sender);
+    function requestDataFeed(
```

Comment:
same as for interface, we don't need to support legacy version, it'll be removed